charts/collector/values.yaml

```diff
@@ -0,0 +1,47 @@
+image:
+  # repository: kong-docker-kong-brain-immunity-base.bintray.io/kong-brain-immunity
+  # tag: latest
+  pullPolicy: IfNotPresent
+  repository: kong-docker-kong-brain-internal-builds.bintray.io/kong-brain
+  tag: master
+
+imagePullSecrets:
+  # - name: bintray-kong-brain-immunity
+  - name: bintray-kong-brain-internal
+
+waitImage:
+  repository: busybox
+  tag: latest
+  pullPolicy: IfNotPresent
+
+
+# Set this to the Kong Admin API hostname
+kongAdminHost: my-kong-kong-admin
+# Set this to the Kong Admin API port
+kongAdminServicePort: 8001
+# Set this to the Kong Admin API protocol
+kongAdminProtocol: http
+containerPort: 5000
+# Set a nodePort which is available
+nodePort: 31555
+service: 
+  type: NodePort
+  port: 5000
```

Comment:
Do you foresee a possibility of having more services in this chart?

If yes then I'd recommend to nest these properties under `collector` or whatever name of the service you want to be.

This is from the experience of Kong's chart where the default or root level properties are mostly for the proxy service and then all other services are nested under the service name.

Reply:
AFAIK there may be another subchart for a TSDB but it seems unlikely another service will be included.

Reply:
I'd still recommend that you indent the these root setting under a property name of the service. That provides a nice API for the end user when they are editing this file or using `--set sname.nodePort=x`.

Reply:
Also, because you have multiple k8s services as part of the chart installation, it is better to clarify this and set the right expectation as to what change will happen.